single-user-dkrz-wps/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM quay.io/jupyter/minimal-notebook:2024-04-19
+
+# Install necessary Python packages using pip
+RUN pip install --no-cache-dir \
+    numpy \
+    pandas \
+    xarray \
+    intake \
+    intake-esm \
+    folium \
+    hvplot \
+    requests \
+    ipywidgets \
+    aiohttp \
+    pyproj \
+    pyparsing \
+    zarr \
+    fsspec \
+    geopy \
+    rooki \
+    cartopy \
+    xmip \
+    esgf-pyclient \ 
+    backports.tarfile \
+    pyvista \
+    pyvista[jupyter]
+
+# Copy sample notebooks to the working directory
+COPY notebooks/ /home/jovyan/work/
```

Comment:
This will not be available at the server as this directory will be mounted from a persistent volume.

If you need to have them available, we should find an alternative way for that